tests/test_dataframe.py

```diff
@@ -0,0 +1,90 @@
+"""this module contains tests for the DataFrame class"""
+
+import numpy as np
+import pytest
+from deeporigin.data_hub import api
+from deeporigin.data_hub.dataframe import DataFrame
+
+from tests.utils import clean_up_test_objects
+
+TEST_PREFIX = "tc-x1zrncgf6F-"
+TEST_DB_NAME = TEST_PREFIX + "db"
+COLUMNS = ["float", "integer"]
+
+
+@pytest.fixture(scope="session", autouse=True)
+def config(pytestconfig):
```

Comment:
this fixture creates a database that we can pull/operate on/write to using the DataFrame class